deploy/heroku-deploy/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN useradd -u $buildkite_agent_uid -m panorama
 
 # Make the platform API gem available to ruby scripts run inside the container.
 # Useful for scripting things like environment variable updates
-RUN gem install platform-api --no-rdoc --no-ri --version "2.1.0"
+RUN gem install platform-api --no-document
```

Comment:
The version of Rubygems in Ruby 2.5 is so old that it was still using `--no-rdoc` and `--no-ri` as flags. Newer versions just use `--no-document`; the old flags are no longer recognized and throw errors.